$NetBSD: patch-argp-parse.c,v 1.1 2023/03/12 12:38:56 js Exp $

Remove __THROW that might be defined to __attribute__((nothrow)) and would thus
cause the compilation to fail.

--- argp-parse.c.orig	2003-12-11 08:37:05.000000000 +0000
+++ argp-parse.c
@@ -1277,13 +1277,13 @@ weak_alias (__argp_input, _argp_input)
 /* Defined here, in case a user is not inlining the definitions in
  * argp.h */
 void
-__argp_usage (__const struct argp_state *__state) __THROW
+__argp_usage (__const struct argp_state *__state)
 {
   __argp_state_help (__state, stderr, ARGP_HELP_STD_USAGE);
 }
 
 int
-__option_is_short (__const struct argp_option *__opt) __THROW
+__option_is_short (__const struct argp_option *__opt)
 {
   if (__opt->flags & OPTION_DOC)
     return 0;
@@ -1297,7 +1297,7 @@ __option_is_short (__const struct argp_o
 }
 
 int
-__option_is_end (__const struct argp_option *__opt) __THROW
+__option_is_end (__const struct argp_option *__opt)
 {
   return !__opt->key && !__opt->name && !__opt->doc && !__opt->group;
 }
